src/stream/src/executor/aggregation/agg_state.rs

```diff
@@ -176,17 +193,17 @@ impl<S: StateStore> AggState<S> {
             }
         };
 
-        let result_row = Row::new(
-            self.group_key
-                .as_ref()
-                .unwrap_or_else(Row::empty)
-                .values()
-                .chain(curr_outputs.iter())
-                .cloned()
-                .collect(),
-        );
-        self.prev_outputs = Some(curr_outputs);
+        let result_row = self
+            .group_key
+            .as_ref()
+            .unwrap_or_else(Row::empty)
+            .concat(curr_outputs.clone().into_iter());
```

Comment:
```suggestion
            .concat(curr_outputs.iter().cloned());
```

Reply:
Same for other occurrences. :)

Reply:
No other occurrence (

Reply:
Oops. BTW, the `into_iter()` can also be eliminated.